components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/admin/v1/dto/KeyManagerPermissionsDTO.java

```diff
@@ -0,0 +1,135 @@
+package org.wso2.carbon.apimgt.rest.api.admin.v1.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import javax.validation.constraints.*;
+
+
+import io.swagger.annotations.*;
+import java.util.Objects;
+
+import javax.xml.bind.annotation.*;
+import org.wso2.carbon.apimgt.rest.api.common.annotations.Scope;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import javax.validation.Valid;
+
+
+
+public class KeyManagerPermissionsDTO   {
+  
+
+    @XmlType(name="PermissionTypeEnum")
+    @XmlEnum(String.class)
+    public enum PermissionTypeEnum {
+        PUBLIC("PUBLIC"),
+        ALLOW("ALLOW"),
+        DENY("DENY");
+        private String value;
+
+        PermissionTypeEnum (String v) {
+            value = v;
+        }
+
+        public String value() {
+            return value;
+        }
+
+        @Override
+        public String toString() {
+            return String.valueOf(value);
+        }
+
+        @JsonCreator
+        public static PermissionTypeEnum fromValue(String v) {
+            for (PermissionTypeEnum b : PermissionTypeEnum.values()) {
+                if (String.valueOf(b.value).equals(v)) {
+                    return b;
+                }
+            }
+return null;
```

Comment:
Seems to be an indentation issue here

Reply:
Fixed in b1f1bcef30dd134cd6c4c26aa35347fab855e066